### PR TITLE
Remove titan dependency single engine

### PR DESCRIPTION
**Pull request by fppt**

### CODE REVIEW

File: mindmaps-migration/sql/src/test/java/io/mindmaps/migration/sql/SQLSchemaMigratorTest.java [hunk 1]
```diff
@@ -98,7 +98,6 @@ public void usersTest() throws SQLException {
     public void alterTableTest() throws SQLException {
         Connection connection = Util.setupExample("alter-table");
         migrator.configure(connection).migrate(loader);
-        graph.rollback();
```
Comment (aelred) [inline mindmaps-migration/sql/src/test/java/io/mindmaps/migration/sql/SQLSchemaMigratorTest.java]: Is this definitely OK? Might want to confirm with Alex.

File: mindmaps-test/pom.xml [hunk 2]
```diff
@@ -42,6 +46,21 @@
             </exclusions>
         </dependency>
         <dependency>
+           <groupId>io.mindmaps</groupId>
+           <artifactId>mindmaps-titan-factory</artifactId>
```
Comment (aelred) [inline mindmaps-test/pom.xml]: Shouldn't this be in `titan-test/pom.xml`?

File: mindmaps-test/titan-test/src/test/java/io/mindmaps/core/ConcurrencyTest.java [hunk 3]
```diff
@@ -55,7 +55,7 @@
 
     @BeforeClass
     public static void startController() throws Exception {
-        startTestEngine();
+        IntegrationUtils.startTestEngine();
```
Comment (aelred) [inline mindmaps-test/titan-test/src/test/java/io/mindmaps/core/ConcurrencyTest.java]: Aw :(

File: mindmaps-test/pom.xml (hunk 2 above)
Comment (fppt) [inline mindmaps-test/pom.xml]: Will fix in next PR

File: mindmaps-test/titan-test/src/test/java/io/mindmaps/core/ConcurrencyTest.java (hunk 3 above)
Comment (fppt) [inline mindmaps-test/titan-test/src/test/java/io/mindmaps/core/ConcurrencyTest.java]: OOps

**Pull request merged**